test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +42,42 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() throws Exception {
+
+        // no items
+        assertHasNull();
+
+        // one item
+        assertHasNull("");
+        assertHasNull("1234");
+        assertHasNull("abcd");
+
+        // multiple items
+        assertHasNull(1, 2, 3);
+        assertHasNull("abc", "def", "ghi");
+        assertHasNull(1, "abc");
+
+        // one null item
+        assertNoNull(1, null);
+        assertNoNull("abc", null);
+        assertNoNull(1, "abc", null);
+
+        // multiple null items
+        assertNoNull(null, null);
+        assertNoNull(null, null, null);
+        assertNoNull(1, 2, null, null);
+        assertNoNull("abc", "def", null, null);
+        assertNoNull(null, 1, null);
+        assertNoNull("abc", null, "abc", null);
+    }
+
+    private void assertNoNull(Object... items) {
```

Comment:
This method is probably not needed since it is a one-liner. All instances of `assertNoNull` can probably be replaced with `assertTrue(Utils.isAnyNull(items));` itself.

Same goes for `assertHasNull`, but of course it is just a matter of preference.